File: 코딩 기초 트레이닝/무작위로 K개의 수 뽑기-p181858/무작위로 K개의 수 뽑기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181858

 def solution(arr, k):
     answer = []

     for a in arr:
         if not a in answer:
             answer.append(a)

         if len(answer) == k:
             break

     if len(answer) < k:
         while len(answer) < k:
             answer.append(-1)

     return answer

File: 코딩 기초 트레이닝/무작위로 K개의 수 뽑기-p181858/무작위로 K개의 수 뽑기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181858

function solution(arr, k) {
    const answer = [];
    let i = 0;
    while(answer.length < k) {
        if(!answer.includes(arr[i])) {
            answer.push(arr[i]);
        }
        i++;

        if(i === arr.length && answer.length < k) {
            break;
        }
    }

    if(answer.length < k) {
        while(answer.length < k) {
            answer.push(-1);
        }
    }

    return answer;
}
